blockmanager.go

```diff
@@ -1738,29 +1709,27 @@ func (b *blockManager) handleNotifyMsg(notification *blockchain.Notification) {
 		bestHeight := band.BestHeight
 		blockHeight := int64(block.MsgBlock().Header.Height)
 		reorgDepth := bestHeight - (blockHeight - band.ForkLen)
-		if b.server.rpcServer != nil &&
-			blockHeight >= b.server.chainParams.StakeValidationHeight-1 &&
+		if blockHeight >= b.cfg.ChainParams.StakeValidationHeight-1 &&
```

Comment:
This should only happen when the `rpcServer` is non nil.  That condition should not be removed.